### PR TITLE
Showing usage message when no arguments are passed 

### PR DESCRIPTION
**Pull request by kapilkd13**
When we run cwltool without any argument we get a "try -h" message. Instead of that directly show Usage message.

### CODE REVIEW

File: cwltool/main.py [hunk 1]
```diff
@@ -630,7 +631,8 @@ def main(argsl=None,  # type: List[str]
                 setattr(args, "workflow", "CWLFile")
             else:
                 _logger.error("")
-                _logger.error("CWL document required, try --help for details")
+                _logger.error("CWL document required, No input file was passed\n")
```
Comment (mr-c) [inline cwltool/main.py]: Is the `\n` really needed?

I would say "CWL document required, no input file was provided"

**Comment by mr-c** [comment]
Jenkins, okay to test

File: cwltool/main.py (hunk 1 above)
Comment (kapilkd13) [inline cwltool/main.py]: Not really, should I remove it

File: cwltool/main.py [hunk 2]
```diff
@@ -432,6 +432,7 @@ def load_job_order(args, t, stdin, print_input_deps=False, relative_deps=False,
         if toolparser:
             if args.tool_help:
                 toolparser.print_help()
+
```
Comment (mr-c) [inline cwltool/main.py]: `cwltool/main.py:435:1: W293 blank line contains whitespace`
from https://travis-ci.org/common-workflow-language/cwltool/jobs/207374774#L164

**Comment by kapilkd13** [comment]
updated message, removed space

**Comment by mr-c** [comment]
Thanks!

**Pull request merged**